E1a-ARSE/ALIN/README.md

```diff
@@ -0,0 +1,174 @@
+
+The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+:zap: **This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test** :zap:
+
+## Itemised Roots
+*This section lists the primary entities and concept templates being validated by this test instruction, these are listed to understand the scope of the data entities being addressed. this is the documentation of the data placed within BIMQ*
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+:construction: under construction :construction:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+- *IfcAlignment*
+- *IfcAlignmentHorizontal*
+- *IfcAlignmentVertical*
+- *IfcAlignmentSegment*
+- *IfcAlignmentHorizontalSegment*
+- *IfcAlignmentVerticalSegment*
+- *IfcAxis2PlacementLinear*
+- *IfcAxis2Placement2D*
+- *IfcAxis2Placement3D*
+- *IfcCircle*
+- *IfcClothoid*
+- *IfcCompositeCurve*
+- *IfcCurveSegment*
+- *IfcLine*
+- *IfcLinearPlacement*
+- *IfcLocalPlacement*
+- *IfcGeometricRepresentationContext*
+- *IfcGeometricRepresentationSubContext*
+- *IfcGradientCurve*
+- *IfcMapConversion*
+- *IfcPolyline*
+- *IfcProject*
+- *IfcProjectedCRS*
+- *IfcProductDefinitionShape*
+- *IfcSite*
+- *IfcShapeRepresentation*
+
+</details>
+
+<details><summary>Concept Templates</summary>
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+- *Alignment Layout*
+- *Alignment Geometry*
+- *Alignment Geometry Gradient*
+- *Project Global Positioning*
+- *Spatial Containment*
+
+</details>
+
+## Test Case Imports
+Test instructions are defined with a modular approach to reduce repetition of validation criteria and test content, and improve vendors ability to solve issues and bugs. therefore this test instruction *imports/reuses* the following Test instructions and entities with the relevant associated validation criteria.
+
+:construction: under construction :construction:
+
+<details><summary>Imports & Reuses</summary>
+
+| TI Code                                  | Test Instruction Title  | Comments                                                     |
+| ---------------------------------------- | ----------------------- | ------------------------------------------------------------ |
+| [IFC4.3AbRV_E0_SSRD](../../E0-SSRD/SSRD) | Spatial Structures Road | Spatial structure for road incuding the dependencies (E0_SSSI, E0_MSTP) |
+
```

Comment:
Do we have a georeferencing TI?

Reply:
Thanks for commenting @pjanck! I have assumed that georeferencing is part of E0_MSTP (if you take a look into the TI dependency diagram from @AlexBrad1eyCT) on Miro https://miro.com/app/board/o9J_lkyDq70=/.
As you say, there's a lot still to be defined. My main issues are:
- I have assumed straight line, circular arc, clothoid for horizontal and straight line, parabolic or circular arc for vertical. Is that all for a road alignment or are there other more obscure cases?
- Is it enough to test straight, left turn, right turn in horizontal and straight up/down (connected with arcs) for vertical?
- How to produce the actual test case? I looked at e.g. the alignment from MCON-2 but that does not contain clothoids. It would be nice to have a set case that uses all necessary segment types in all typical ways (left/right, up/down, transitions, ...).

Adding you to the list of reviewers :-)

Reply:
For a road alignment, even parabolic arc is rare enough to raise the question: is it going to be mandatory for certification? Looking at validation criteria "For this test case to be considered passed **all capabilities** listed in this section shall be verified, with no exception" and "At least 1 instance of each entity listed in Itemised Roots is present in the file", I'd be very careful with what is included in the case (should be testing IFC export/import, not software capability in general). If it's not against any chosen certification strategy (?) might be reasonable to allow also "passed with exceptions (documented)", but if the validation criteria for a case really must be that draconian, I'd suggest making first a basic road alignment case with linear, circular arc and clothoid for horizontal; linear and circular arc for vertical.

Reply:
During my "peak days", as a developer of road design software (this was a long time ago), the only vertical "curved curve" used was parabolic arc. Unfortunately, I didn't find anything regarding this in the RA. Might have missed it though.  I wanted to mention both circular and parabolic until more info available

Reply:
@larswik I'm not saying that parabolic arc wouldn't be valid vertical curve on roads and shouldn't be included in some test case. I'm just asking does it have to be supported as a mandatory requirement? If a software doesn't do parabolic arcs since their users do not use them they should still be able to get certified - I hope.

Reply:
@JuhaHy good point!

Reply:
Meta: I'd move this discussion to a separate issue for easier traceability. Searching for these comments in the future will be a pain, as well as a separate issue enables others to chime in as well. @larswik I'll copy the comments if that is ok with you.

> georeferencing is part of E0_MSTP which is mentioned

That answers my question and resolves this conversation.